app/Providers/AppServiceProvider.php

```diff
@@ -31,10 +32,10 @@ public function register()
     public function boot()
     {
         Context::initialize(
-            env('SHOPIFY_API_KEY'),
-            env('SHOPIFY_API_SECRET'),
-            env('SCOPES'),
-            str_replace('https://', '', env('HOST')),
+            Config::get('shopify.api_key'),
+            Config::get('shopify.api_secret'),
+            Config::get('shopify.scopes'),
+            str_replace('https://', '', Config::get('app.url')),
```

Comment:
Just double-checking: will this be set to the same value as the `HOST` in the env file? It will have to match that for the CLI to work properly with it.